lib/src/source_visitor.dart

```diff
@@ -2022,7 +2022,7 @@ class SourceVisitor extends ThrowingAstVisitor {
     _simpleStatement(node, () {
       token(node.libraryKeyword);
       space();
-      visit(node.name);
+      visit(node.name2);
```

Comment:
what's happening with this one?

Reply:
In analyzer 5.1.0, `.name` is deprecated in favor of `.name2`.

Reply:
Is this on a different type than the rest of the invocations?

Reply:
Oops, my bad. In all of the _other_ changes, `.name2` is deprecated in favor of `.name`.

For this change, `LibraryDirective.name`, a non-nullable String is being replaced with `LibraryDirective.name2`, a nullable String (for the case of an omitted named).